### PR TITLE
Fix clem

### PR DESCRIPTION
**Pull request by KateMoreva**
Only for S3

### CODE REVIEW

**Comment by constantinpape** [comment]
Hey @tischi,
this should fix https://github.com/mobie/mobie-io/issues/93. Could you have a look and if all seems correct merge it and make a new MoBIE-beta release with the changes, so that the clem project works again?
Thanks!

File: src/main/java/org/embl/mobie/viewer/MoBIE.java [hunk 1]
```diff
@@ -371,23 +372,29 @@ public synchronized ImageSource getSource( String sourceName )
...
 		try
 		{
 			SpimData spimData = tryOpenSpimData( imagePath, imageDataFormat );
 			sourceNameToImgLoader.put( sourceName, spimData.getSequenceDescription().getImgLoader() );
 
 			final SourceAndConverterFromSpimDataCreator creator = new SourceAndConverterFromSpimDataCreator( spimData );
 			SourceAndConverter< ? > sourceAndConverter = creator.getSetupIdToSourceAndConverter().values().iterator().next();
-			// Touch the source once to initiate the cache,
-			// as this speeds up future accesses significantly
-			sourceAndConverter.getSpimSource().getSource( 0,0 );
```
Comment (tischi) [inline src/main/java/org/embl/mobie/viewer/MoBIE.java:390]: @KateMoreva Why did you remove this? This is important for the performance.

**Review by tischi — COMMENTED** [review]
Looks good to me. 
I had one important comment.

File: src/main/java/org/embl/mobie/viewer/MoBIE.java (hunk 1 above)
Comment (KateMoreva) [reply src/main/java/org/embl/mobie/viewer/MoBIE.java:390]: Oh, it is probably merge mistake.

Comment (KateMoreva) [reply src/main/java/org/embl/mobie/viewer/MoBIE.java:390]: @tischi, should be fixed

**Comment by tischi** [comment]
@KateMoreva 

There are test failures that could be related to the changes. 
Could you please have a look?

https://github.com/mobie/mobie-viewer-fiji/runs/6173433055?check_suite_focus=true#step:8:399

File: src/main/java/org/embl/mobie/viewer/MoBIE.java [hunk 2]
```diff
@@ -116,7 +116,7 @@ private MoBIESettings setImageDataFormat( String projectLocation )
 
 		if ( imageDataFormat.size()  != 0 )
 		{
-			if ( ! project.getImageDataFormats().contains( imageDataFormat ) )
+			if ( project.getImageDataFormats().stream().noneMatch(imageDataFormat::contains) )
```
Comment (tischi) [inline src/main/java/org/embl/mobie/viewer/MoBIE.java:119]: wow, fancy :-)
